specification.md

```diff
@@ -301,7 +301,7 @@ we can define a state filter:
 ```json
 {
   "stateDataFilter": {
-    "input": "${ {fruits: .fruits} }"
+    "input": "${fruits: .fruits}"
```

Comment:
this is wrong imo. the original expression its is clear that the expression returns an object
in changex expression its not even an expression that can be evaluated by an expression parser
it defines really its own expression syntax imo 

if .fruits returns an array in some case then what you create an element fruits which is of array type? but thats not the user wanted he wants to just have the array. confusing imo.

Reply:
thx, that is wrong